src/index.ts

```diff
@@ -111,13 +116,20 @@ app.get('/questions', async (req: Request, res: Response) => {
 		const questions = data.questions;
 		const category = req.query.category;
 		const difficulty = req.query.difficulty;
+		const favourite = parseBoolean(req.query.favourite);
```

Comment:
think this should be `const favourite = parseBoolean(req.query.favourited)` as the key in the object is `favourited`